doc/release-scripts/Fix_For_MultipleTagID_Toggler_MOdf.py

```diff
@@ -0,0 +1,38 @@
+
+'''Issue: In a particular activity page, there are multiple transcripts and on click of the later has effect on the former but not on the corresponding
+   Reson: Both the transcripts are having the same ID 
+  Unit : English Beginner Lesson Name : Places Around Us Acitity page :Warm Up
+
+ Fix : Have modified the first encountered ID as Toggler09
+
+'''
+
+from gnowsys_ndf.ndf.models import node_collection
+import re
+
+''' To collect all the activities under the English module'''
+GSystemnds = node_collection.find({
+            '_type':'GSystem',
+            'member_of':[ObjectId('5752ad552e01310a05dca4a1')],
+            'group_set':{'$in':[
+                    ObjectId('5943ff594975ac013d3701fc'),ObjectId('5943fd564975ac013d36fdae'),
+                    ObjectId('59425be44975ac013cccb909'),ObjectId('59b6565c2c47960148218050')]},
+            'collection_set':[]})
```

Comment:
## Never ever add hard-coded/database things in code file, unless it is an part of factory (which ensures it's presence in DB) or only in special conditions or it's not possible to derive.
- Take help of one more query to derive expected value / `_id` / `name` in query.